{
 "bad": [
  "ConfigParser",
  "ConfigParser.NoOptionError",
  "ConfigParser.NoSectionError",
  "ConfigParser.SafeConfigParser",
  "errno",
  "errno.ENOENT",
  "from_keywords_f",
  "from_vcs_f",
  "get",
  "get_keywords_f",
  "os.getcwd",
  "os.path.abspath",
  "os.path.basename",
  "os.path.dirname",
  "os.path.exists",
  "os.path.join",
  "os.path.normcase",
  "os.path.realpath",
  "os.path.relpath",
  "os.path.splitext",
  "os.uname",
  "os.unlink",
  "py2exe.build_exe.py2exe",
  "py2exe.build_exe.py2exe.run",
  "py2exe.distutils_buildexe.py2exe",
  "re.M",
  "re.S",
  "splauncher.__init__.get_versions",
  "sys",
  "sys.argv",
  "sys.exc_info",
  "sys.exit",
  "sys.modules",
  "sys.platform",
  "sys.stderr",
  "sys.version_info"
 ],
 "deps": {
  "ConfigParser": [],
  "__future__": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "configparser": [
   "configparser/conda-forge/noarch/configparser-5.0.0-py_0",
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "cx_Freeze": [
   "cx_freeze/conda-forge/linux-64/cx_freeze-6.0-py38h516909a_0"
  ],
  "datetime": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "distutils": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "drmaa": [
   "drmaa/conda-forge/noarch/drmaa-0.7.9-py_0"
  ],
  "errno": [],
  "from_keywords_f": [],
  "from_vcs_f": [],
  "get": [],
  "get_keywords_f": [],
  "json": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "logging": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "os": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "py2exe": [],
  "re": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "setuptools": [
   "setuptools/conda-forge/linux-64/setuptools-41.2.0-py27_0"
  ],
  "splauncher": [],
  "subprocess": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "sys": []
 }
}